regression-test/framework/src/main/groovy/org/apache/doris/regression/suite/SuiteCluster.groovy

```diff
@@ -302,7 +316,20 @@ class SuiteCluster {
             cmd += ['--fe-follower']
         }
 
-        cmd += ['--wait-timeout', String.valueOf(180)]
+        if (options.sqlModeNodeMgr) {
+            cmd += ['--sql-mode-node-mgr']
+        }
+        if (options.beMetaServiceEndpoint) {
```

Comment:
f (!options.beMetaServiceEndpoint) {
   cmd += ['--no-be-metaservice-endpoint']
}